src/components/home/index.js

```diff
@@ -35,7 +35,7 @@ const Home = () => {
       }}
     >
       {dev && <UserProfile />}
-      {dev && <SearchBox />}
+      {!dev && <SearchBox />}
```

Comment:
We need to test it out behind dev flag first. Reverting this line should solve the issue.

Reply:
but then our search box is not showing

Reply:
You need to add `?dev=true` after the url for it to show.

Reply:
ok got it :)